src/providers/database.ts

```diff
@@ -369,7 +369,7 @@ export class DataSnapshot {
     private app?: firebase.app.App,
     instance?: string
   ) {
-    if (app && app.options.databaseURL.startsWith('http:')) {
+    if ( app && app.options && app.options.databaseURL && app.options.databaseURL.startsWith('http:') ) {
```

Comment:
I actually think we can use optional chaining. Since we are using Typescript and targeting ES5, tsc will take care of translating the optional chaining into something that's supported in the runtime that does not support it.

Reply:
Hi Daniel

Thanks for pointing that out!
I've updated database.ts:372 to use optional chaining.

I've also added an entry to the CHANGELOG.md 
Please let me know if this is sufficient, or requires more detail in the changelog.